entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -l
+
+cp /evaluator.js /github/workspace
+npm install -g jest
+jest --json --outputFile=evaluation.json
+node evaluator.js evaluation.json mapping.json result.json
+
+if [ $? != 0 ]; then
```

Comment:
What is this conditional for?

Reply:
To set actions exit code

- https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#setting-an-actions-status-using-exit-codes